msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/PermissionLevelManagerImpl.java

```diff
@@ -913,6 +921,7 @@ private PermissionsMask getDefaultReviewerPermissionsMask() {
 		  mask.put(PermissionLevel.READ, Boolean.valueOf(true));
 		  mask.put(PermissionLevel.MARK_AS_READ,Boolean.valueOf(true));
 		  mask.put(PermissionLevel.MODERATE_POSTINGS, Boolean.valueOf(false));
+		  mask.put(PermissionLevel.IDENTIFY_ANON_AUTHORS, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.DELETE_OWN, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.DELETE_ANY, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.REVISE_OWN, Boolean.valueOf(false));
```

Comment:
You can use the builder pattern to make more readable code:

    PermissionMaskBuilder builder = new PermissionMarkBuilder(); // Default everything to false
    PermissionMask mask = builder.addRead().addMarkRead().addDeleteOwn().build();

I'm not suggesting you need todo this here, just suggesting it as a possible coding style solution.